assignments/#10.1.py

```diff
@@ -0,0 +1,34 @@
+"""
+This program split given large number into hundreds, tens and units.
+"""

```

Comment:
This format is called _**docstring**_.  If you put multline string (triple-quotes) at the beginning of the file, you can access it from many places. See line 18 for example